frontend/public/files/cdogs.postman_collection.json

```diff
@@ -0,0 +1,147 @@
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "https://cdogs-master-idcqvl-prod.pathfinder.gov.bc.ca/v1/docGen",
```

Comment:
This applies to all the urls in each collection - should we point them at dev?  That will be the first place they have access, and they may never promote to prod.

Reply:
Good point for now definitely, just thinking about this opens up some stuff though... maybe should discuss in the new year. If we have active unreleased changes in Dev master would we want new onboarders looking at that? Maybe test? Or should we only merge down to master once we tag a formal release.
Probably should get everyone together to discuss our final release tagging policy for that.

Changing to dev for now though as your point stands.

Reply:
Yeah... would seem we need a stable, versioned site/app for the demos... can you add a ticket to the backlog?  I think that will tie into cleaning up the URLs etc, the final polish.